resolver-dns/src/main/java/io/netty/resolver/dns/SingletonDnsServerAddresses.java

```diff
@@ -30,14 +29,23 @@ public InetSocketAddress next() {
         }
 
         @Override
+        public int size() {
+            return 1;
+        }
+
+        @Override
+        public DnsServerAddressStream duplicate() {
+            return this;
+        }
+
+        @Override
         public String toString() {
             return SingletonDnsServerAddresses.this.toString();
         }
     };
 
     SingletonDnsServerAddresses(InetSocketAddress address) {
         this.address = address;
-        strVal = new StringBuilder(32).append("singleton(").append(address).append(')').toString();
```

Comment:
@normanmaurer - I got rid of this member variable to make `SingletonDnsServerAddresses` more consistent with other implementations of `DnsServerAddresses` and it may not be necessary during regular operations. I can pull this into another PR if you prefer but figured while I was in here I would just make the change.